tests/TestCase/Controller/Component/AuthorizationComponentTest.php

```diff
@@ -104,4 +103,50 @@ public function testAuthorizeCustomException()
         $article = new Article(['user_id' => 99]);
         $this->Auth->authorize($article);
     }
+
+    public function testAuthorizeModelSuccess()
+    {
+        $service = new AuthorizationService(new OrmResolver());
+        $identity = new IdentityDecorator($service, ['can_edit' => true]);
+        $this->Controller->request = $this->Controller->request
+            ->withAttribute('identity', $identity);
+
+        $this->Auth->authorizeModel();
+        $this->assertTrue(true);
```

Comment:
This doesn't look right.

Reply:
I’m not sure how to assert that exception has not been thrown.

Reply:
You could assert the return of `authorizeModel`

Reply:
What about `$this->assertNull($this->Auth->authorizeModel());`? Adds to assertion count, allows further assertions, and will fail if an exception is thrown.